### PR TITLE
Add categories filter

### PR DESCRIPTION
**Pull request by filipebfernandes**

### CODE REVIEW

**Comment by jfranciscosousa** [comment]
@filipebfernandes look into `lodash-es` we can use that.

File: src/components/CategoriesFilter/index.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+  render() {
+    const { selectedCategories } = this.state;
+
+    return (
+      <div styleName="root">
+        <FormControl styleName="form-control">
+          <InputLabel
+            htmlFor="select-multiple-chip"
+            style={{ fontSize: "18px" }}
```
Comment (jfranciscosousa) [inline src/components/CategoriesFilter/index.js]: Why a style object? Stylename doesn't work?

Comment (filipebfernandes) [reply src/components/CategoriesFilter/index.js]: Yes

**Comment by filipebfernandes** [comment]
@jfranciscosousa Check again, please

**Comment by filipebfernandes** [comment]
@jfranciscosousa can I merge this to master now?

**Pull request merged**